theme/style.css

```diff
@@ -9,10 +9,50 @@
   background-position: center;
 }
 
+
+
 .header h1 {
```

Comment:
You don't have a `header` class, so this selector will never match an element. Maybe try `#Allan h1`?